src/DIRAC/Resources/Catalog/RucioFileCatalogClient.py

```diff
@@ -697,3 +701,193 @@ def getDirectorySize(self, lfns, longOutput=False, rawFiles=False):
             except Exception as err:
                 return S_ERROR(str(err))
         return S_OK(resDict)
+
+    @checkCatalogArguments
+    def getFileUserMetadata(self, path):
+        """Get the meta data attached to a file, but also to
+        all its parents
+        """
+        path = next(iter(path))
+        resDict = {"Successful": {}, "Failed": {}}
+        try:
+            did = self.__getDidsFromLfn(path)
+            meta = next(self.client.get_metadata_bulk(dids=[did], inherit=True, plugin="ALL"))
```

Comment:
@fstagni What is the compatibility policy with rucio?

The `plugin=` option was just added in rucio (not even released). Should we check a version here and throw an error like `getFileUserMetadata is only supported with rucio versions >= 36.5` or whenever it is going to be released?

Reply:
cc @bari12

Reply:
It will come with the `37.0.0` release. If we (Well @maxnoe ) wants to do the effort, we can also backport it to the running LTS release lines 35 and 32. But I am not sure if it is worth the effort.

Reply:
At least for us it is fine to have to update to a new rucio version for this